.editorconfig

```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
it appears there are still is still the no newline character in this file :/

Reply:
no newline character?

Reply:
oh weird, its gone now!?

Reply:
![Screen Shot 2019-12-14 at 7 59 17 PM](https://user-images.githubusercontent.com/1368727/70856501-3de75380-1eac-11ea-8d8f-97f0c44acedf.png)

when i go to the full view its there, but inline in the PR conversation tab its not 🤷‍♂

Reply:
@PeteDuncanson do you want to remove this character and then we can merge it? or should I re-create the PR with the fix, and give you credit?